avocado/core/remote/runner.py

```diff
@@ -203,6 +203,7 @@ def run_suite(self, test_suite, mux, timeout, replay_map=None, test_result_total
         fabric_debugfile = os.path.join(self.job.logdir, 'remote.log')
         paramiko_logger = logging.getLogger('paramiko')
         fabric_logger = logging.getLogger('avocado.fabric')
+        remote_logger = logging.getLogger('avocado.remote')
```

Comment:
Not really. I mean paramiko is a library, it will always output to `paramiko`. We can discuss the `avocado.fabric` as avocado creates it, but I think it's another task.

The thing is that the docker should not abuse the `avocado.fabric` logger, that would be clearly wrong. I don't think we need a specific `docker` logger either, so I created `avocado.remote`. Later we can revise the `avocado.fabric` and move it to the more-specific `avocado.remote`, or into `avocado.remote.fabric` to allow filtering just that.